actioncable/lib/action_cable/connection/base.rb

```diff
@@ -2,7 +2,7 @@
 
 module ActionCable
   module Connection
-    # For every WebSocket the Action Cable server accepts, a Connection object will be instantiated. This instance becomes the parent
+    # For every WebSocket connection the Action Cable server accepts, a Connection object will be instantiated. This instance becomes the parent
```

Comment:
I think specifically specifying the fact that it's on a per connection basis helps with understanding -- does "For every WebSocket" == "For every WebSocket socket/connection"? I just thought that WS was the name of the protocol / specification 😬